app/views/users/show.html.erb

```diff
@@ -1,24 +1,136 @@
-<p class="fs-5 mt-4 mb-2">
-  <strong>Név:</strong>
-  <%= @user.name %>
-</p>
+<div class="row">
+  <div class="col-12">
+    <div class="card">
+      <div class="card-header text-center">
+        <span class="text-muted">
+          <svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" fill="currentColor" class="bi bi-person-circle" viewBox="0 0 16 16">
+          <path d="M11 6a3 3 0 1 1-6 0 3 3 0 0 1 6 0z"/>
+          <path fill-rule="evenodd" d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8zm8-7a7 7 0 0 0-5.468 11.37C3.242 11.226 4.805 10 8 10s4.757 1.225 5.468 2.37A7 7 0 0 0 8 1z"/>
+        </svg>
```

Comment:
I like heroicons better, but this can stay too
I recommend user-circle on heroicons

Reply:
Well, the best practice would be to use some library (Bootstrap icons, FontAwesome ...), and change out all that ugly ass `<svg>` tags to a cleaner syntax: i.e `<i class="bi bi-person-circle"></i>`.

I created an issue about it, but not will be in v1.0 milestone. see #140